### ISSUE TITLE
A11y_Visual Studio Code Client_Sep2021_Debug Adapter Protocol_HomePage_Keyboard: Keyboard focus is not visible on Selected link  Star and Startgazers after pressing Tab key..

### ISSUE THREAD

**Issue by harshgangrade111** [microsoft/debug-adapter-protocol#206]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTag:**#A11yTCS;#DesktopWeb;#Win10;#WCAG2.4.7;#STP;#FTP;#A11yMAS;#ChromiumEdge;#Keyboard;#A11ySev1;#P2_Visual Studio Code Clien_Sep2021;#Visual Studio Code Client;#P2_WCP;#AILimited;

## **Environment Details:**

Application Name: Visual Studio Code Client
#URL: https://microsoft.github.io/debug-adapter-protocol
Browser Versions: Microsoft Edge Version 93.0.961.38 (Official build) (64-bit)
Windows Version: Windows10

**Example Page URL**: https://microsoft.github.io/debug-adapter-protocol

## **Repro Steps:**
1.	Open the URL: https://microsoft.github.io/debug-adapter-protocol
2.	Debug Adapter Protocol home screen will get open.
3.	Tab to Links Start and stargazers present under main heading "Debug Adapter Protocol" and check focus is visible or not.

## **Actual:**

Keyboard focus is not visible on Selected link  Star and Start gazers after pressing Tab key.
**Same issue is also observed throughout the the application where similar links present.**

## **Expected:**

Keyboard focus outline should be properly visible on selected links after pressing tab key when focus is on it.

## **User Impact:**

Low vision users and keyboard users will not be able to see focusable control.

## **Attachment:**
![#206](https://user-images.githubusercontent.com/60422426/133749784-e6b2627b-9f00-4925-afee-eda933cccd4a.gif)

**Comment by weinand** [comment]
@sana-ajani thanks a lot for fixing this.
Can we close this issue now?

**Comment by rzhao271** [comment]
I'm unable to verify that the fix works. I opened a new issue and assigned myself at https://github.com/microsoft/debug-adapter-protocol/issues/253

**Comment by NiharikaTipirishetty** [comment]
#P2_debugadapterprotocol_Web_Sep2021;

**Comment by prabhudevu** [comment]
@rzhao271 Bugs can be closed after verification or valid justification by trusted testers only. As per your comment on bug: https://github.com/microsoft/debug-adapter-protocol/issues/253 you have mentioned it as won't fix since it is dependent onon a third party. In this case depending on the component being a third-party library or product external bugs are logged. Requesting you to please re open this bug and let us know the third-party component.

**Comment by rzhao271** [comment]
For the star buttons we're using https://github.com/ntkme/github-buttons. The feature can be previewed at https://buttons.github.io/.

**Comment by rzhao271** [comment]
Filed an upstream issue https://github.com/ntkme/github-buttons/issues/287

**Comment by rzhao271** [comment]
Upstream issue fixed. The issue is ready to close again after verification.

**Comment by isidorn** [comment]
Closing since upstream issue fixed. And testers can verify and let us know.

**Comment by fsteffi** [comment]
GitHubTag: #External;#External:GitHub;#ThirdParty;


external bug:
[https://github.com/ntkme/github-buttons/issues/287](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fntkme%2Fgithub-buttons%2Fissues%2F287&data=05%7C01%7Cv-stfe%40microsoft.com%7Cfffdbe2a637546a2c2de08da755b1ba0%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637951331435076922%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Ns%2FZmwUx0IdPWnM34zFS19GCoVuP3F%2FEuxQydmdoUxs%3D&reserved=0)

**Comment by BhanuPrakashGit** [comment]
GithubTags: #Closed;
